certified-connectors/U.S. Bank Treasury Management/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2778 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "U.S. Bank Treasury Management",
+    "description": "U.S. Bank payments and treasury banking APIs.",

```

Comment:
@jzworkman thank you so much for fixing the connector files. Unfortunately, the word "API" is not allowed in the description too. Please see [here ](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info)and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector). 
Thank you so much again.